backend/emailSystem.py

```diff
@@ -0,0 +1,34 @@
+import smtplib, ssl
+
+port = 465 #For SSL
+password = ""#input("password: ") #we can add the actual password here instead
+
+# Create a secure SSL context
+context = ssl.create_default_context()
+
+#EMAIL
+sender_email = "goldchest.steam@gmail.com" #our email needs to be changed
+reciever_email = "samuel.jothimuthu@gmail.com" #sender email will need to be updated
+message_test = """\
```

Comment:
Should be dynamic with a placeholder

Reply:
sender_email is basically a constant since that is what we are using. 
It is mostly there for testing purposes. Can you be more specific

Reply:
Oh sorry, I was talking about the message_text variable.